plugins/reporters/trustsource/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Use Unix line endings for expected test results for consistency across platforms.
+src/funTest/assets/*-expected-*	text eol=lf
```

Comment:
This file is not required if you don't have any assets for functional tests that are sensitive towards line endings.